### PR TITLE
docs(release-notes): release note changes for 1.2

### PR DESCRIPTION
**Pull request by matt-primrose**
## PR Checklist

- [x] Leverages formatting as provided by https://squidfunk.github.io/mkdocs-material/reference/abbreviations/ 
- [x] Have viewed and verified edits in rendered form ensuring proper formatting

## What are you changing?
Updating release notes for the 1.2 release

### CODE REVIEW

**Review by rsdmike — APPROVED** [review]

File: docs/release-notes.md [hunk 1]
```diff
@@ -1,47 +1,45 @@
...
 
 ### Additions
 #### [RPS](Glossary.md#r)
-- **Network Configuration on Intel&reg; AMT devices:** Added feature to AMT Profiles to select configuring Intel&reg; AMT devices to use DHCP or Static IP.  When set for static IP, Intel&reg; AMT will sync with the Host OS static IP address.  Static IP Address is not supported with CIRA profiles.
-- **MEBx Password configuration:** The AMT Profile now supports configuration options for setting either a defined or randomized MEBx password on Intel&reg; AMT devices.
-- **CORS support:** Added support for Cross-Origin Resource Sharing (CORS).
+- **Hostname and Device Metadata tags:** We have added the ability to define metadata tags for a each Intel&reg; AMT profile.  These tags will be applied to each device that is configured using the profile.  Additionally, if RPC sends up the Intel&reg; AMT device OS hostname during activation, RPS save this information as part of the device information.  When RPS configures a device, it will perform a POST call to MPS to register that device in MPS.  This works in conjunction with the improved metadata gathering that RPC added in 1.1.
```
Comment (bwendlandt-intel) [inline docs/release-notes.md:9]: 'during activation, RPS **will** save this information...'

Comment (bwendlandt-intel) [inline docs/release-notes.md:9]: 'during activation, RPS **will** save this information...'

File: docs/release-notes.md [hunk 2]
```diff
@@ -1,47 +1,45 @@
...
+- **Node 14 LTS support:** RPS has been updated to support running on Node 14 LTS. 
 
 #### [RPC](Glossary.md#r)
-- **Added/updated getting metadata from Intel&reg; AMT device:** Improved getting DNS Suffix (OS), Hostname (OS), FQDN (AMT), and DNS Suffix (AMT) from Host OS and Intel&reg; AMT.
-- **CentOS 7 Support:** Build steps added to support CentOS 7.
-- **--nocertcheck option:** Option added to skip the TLS cert check for Websocket connections. This is to aid testing in development environments where self-signed TLS certificates are used.
+- **Docker support added:** Added a docker file to RPC that allows customers to build and execute RPC in a docker container on the Intel&reg; AMT device.
```
Comment (bwendlandt-intel) [inline docs/release-notes.md:13]: Dockerfile - one word

File: docs/release-notes.md [hunk 3]
```diff
@@ -1,47 +1,45 @@
...
+
 
 #### [MPS](Glossary.md#m)
-- **CORS Support:** Added support for Cross-Origin Resource Sharing (CORS).
+- **PostgreSQL integration added to MPS:** To support additional metadata we are adding to MPS, we have added a PostgreSQL reference implementation to MPS.
+- **Hostname and Device Metadata tags:** MPS will store device hostname and metadata tag information it receives from RPS in it's PostgreSQL DB.  When device information calls are made, the device hostname will be returned to the caller along with the other device information previously provided.  Additionally, the GET devices call now supports an additional tags field (comma-delimited list) that will filter the devices based on the set of tags provided.  This greatly improves scaling of this call with very large number of devices as well as reduces the need for filtering on the caller's side.
```
Comment (bwendlandt-intel) [inline docs/release-notes.md:18]: '...from RPS in **its** PostgreSQL DB.'

File: docs/release-notes.md [hunk 4]
```diff
@@ -1,47 +1,45 @@
...
+#### [MPS](Glossary.md#m) & [RPS](Glossary.md#r)
+- **Deprecation announcement of Developer Mode for both RPS and MPS** We have making lots of strides in improving production mode deployments through the use of deployment scripts and docker containers.  Due to this work, the need for a "simple" developer deployment mode that doesn't use Vault or PostgreSQL is no longer required.  As such, we'll be removing this mode prior to our 1.4 release.  After the removal of developer mode, Vault and PostgreSQL will be required by default but the docker deployment scripts we have will make getting these services up and running easy.  Customers will still have the option to modify the Vault and PostgreSQL interfaces to point to the database service(s) of their choice.
```
Comment (bwendlandt-intel) [inline docs/release-notes.md:33]: 'We have **made** lots of strides in...'

**Review by bwendlandt-intel — CHANGES_REQUESTED** [review]

File: docs/release-notes.md (hunk 1 above)
Comment (matt-primrose) [reply docs/release-notes.md:9]: updated in amended commit

File: docs/release-notes.md (hunk 2 above)
Comment (matt-primrose) [reply docs/release-notes.md:13]: updated in amended commit

File: docs/release-notes.md (hunk 3 above)
Comment (matt-primrose) [reply docs/release-notes.md:18]: updated in amended commit

File: docs/release-notes.md (hunk 4 above)
Comment (matt-primrose) [reply docs/release-notes.md:33]: updated in amended commit

File: docs/release-notes.md (hunk 3 above)
Comment (kimberlyxdavis) [reply docs/release-notes.md:18]: "its PostgreSQL"

**Comment by bwendlandt-intel** [comment]
Did something get overwritten? I don't see any changes made in the latest commit?

File: docs/release-notes.md [hunk 5]
```diff
@@ -1,47 +1,45 @@
...
 
-#### [UI Toolkit](Glossary.md#u)
-- **CORS support:** Added support for Cross-Origin Resource Sharing (CORS).
-- **MEBx Password option added to Profile Editor:** Provides the ability to set a MEBx password when activating Intel&reg; AMT devices.
-- **Network Setting option added to Profile Editor:** Provides the ability to set either DHCP or Static IP setting when activating Intel&reg; AMT devices.
-- **Edit feature added to Profile Editor:** Added ability to edit AMT, CIRA, and Domain profiles in the Profile Editor.
+#### [Sample UI](Glossary.md#s)
+- **New look & feel** The Sample UI has had a full rewrite in the 1.2 release based on much feedback from our customers.  This updated UI supports the new hostname and metadata tag features.  While this is still just a reference implementation to showcase the features of the MPS and RPS components, this restructuring of the Sample UI will:
+    * Aid developers as an example of how to use our APIs 
+    * Help understand the value of Open AMT Cloud Toolkit.
```
Comment (kimberlyxdavis) [inline docs/release-notes.md:25]: I actually wrote something about this in one my wireframe examples.

Use the Sample UI in the pre-deployment phase of solution development to:
-learn how the Open AMT Cloud Toolkit connects development systems to managed devices.
-set up profiles, configurations, and domains.
-understand Open AMT's implementation of the REST APIs and how they can be used in your console application. 

You can use it or not. It's up to you.

File: docs/release-notes.md [hunk 6]
```diff
@@ -1,47 +1,45 @@
...
+#### [RPS](Glossary.md#r)
+- **RPS API changes** We have performed some fundamental restructuring of the RPS REST APIs to better align with industry RESTful API standards.  With these changes you'll see a greatly simplified structure, error messages in proper JSON structure, consistent property names across request/response, and correct HTML status codes being returned.  *This is a breaking change for customers who have already implemented against the 1.1 API.*
+#### [UI Toolkit](Glossary.md#u)
+- **Deprecation announcement of non-socket UI Toolkit components:** We are removing the following UI Toolkit components in the 1.3 release: Audit Log, CIRA Config, Device Grid, Domain, and Profile.  All of these components are just UI implementations of our existing REST APIs and the need for a REACT component for these items is overkill.  We will continue to support and maintain the REACT components for KVM and SOL.  Additionally, we have plans on expanding the KVM and SOL components to support additional UI frameworks beyond REACT.
```
Comment (kimberlyxdavis) [inline docs/release-notes.md:31]: I don't know why but the word "overkill" here...

**Review by kimberlyxdavis — APPROVED** [review]

**Review by bwendlandt-intel — APPROVED** [review]

**Pull request merged**